src/Neo.CLI/CLI/MainService.Tools.cs

```diff
@@ -90,15 +89,44 @@ private void OnParseCommand(string value)
         }
 
         /// <summary>
-        /// Converts an hex value to a big integer
+        /// Little-endian to Big-endian
+        /// input:  ce616f7f74617e0fc4b805583af2602a238df63f
+        /// output: 0x3ff68d232a60f23a5805b8c40f7e61747f6f61ce
         /// </summary>
-        /// <param name="hexString">
-        /// Hexadecimal value to be converted
-        /// </param>
-        /// <returns>
-        /// Returns null when is not possible to parse the hex value to big integer value;
-        /// otherwise, returns the string that represents the converted big integer.
-        /// </returns>
+        [ParseFunction("Little-endian to Big-endian")]
+        private string? LittleEndianToBigEndian(string hex)
+        {
+            try
+            {
+                if (!IsHex(hex)) return null;
+                return "0x" + hex.HexToBytes().Reverse().ToArray().ToHexString(); ;
+            }
+            catch (FormatException)
+            {
+                return null;
+            }
+        }
+
+        /// <summary>
+        /// Big-endian to Little-endian
+        /// input:  0x3ff68d232a60f23a5805b8c40f7e61747f6f61ce
+        /// output: ce616f7f74617e0fc4b805583af2602a238df63f
+        /// </summary>
+        [ParseFunction("Big-endian to Little-endian")]
+        private string? BigEndianToLittleEndian(string hex)
+        {
+            bool hasHexPrefix = hex.StartsWith("0x", StringComparison.InvariantCultureIgnoreCase);
+            hex = hasHexPrefix ? hex[2..] : hex;
+            if (!hasHexPrefix || !IsHex(hex)) return null;
+            return hex.HexToBytes().Reverse().ToArray().ToHexString();
```

Comment:
```suggestion
            var bytes = Convert.FromHexString(hex);
            return Convert.ToHexString([.. bytes.Reverse()]);
```

Reply:
It will crash when parse "0x3ff68d232a60f23a5805b8c40f7e61747f6f61ce"

Reply:
wrap in `TryCatch`